src/WCSimWCTrigger.cc

```diff
@@ -36,10 +36,12 @@ const double WCSimWCTriggerBase::LongTime = 10E6; // ns = 10ms. event time
 
```

Comment:
Are there any changes actually made to WCSimWCTrigger.cc? It looks like this is mostly inserted extra spaces. Can you fix this to just be the new changes required for this pull request?